README-pt-BR.md

```diff
@@ -0,0 +1,347 @@
+A visão geral do _The Turing Way_ começa em decorridos 5:13.
+
+### Sobre o projeto
+
+A pesquisa reproduzível é necessária para garantir que trabalhos científicos sejam confiáveis.
+Financiadores e editores estão começando a exigir que as publicações incluam o acesso aos dados subjacentes e ao código de análise.
+O objetivo é garantir que todos os resultados possam ser verificados de forma independente e aproveitados em trabalhos futuros.
+Às vezes, é mais fácil falar do que fazer.
+Compartilhar esses resultados de pesquisa significa compreender o gerenciamento de dados, o funcionamento das bibliotecas, o desenvolvimento de software e as técnicas de integração contínua: habilidades que não são, de forma ampla, ensinadas ou esperadas de pesquisadores acadêmicos e cientistas de dados.
+Como essas atividades não são comumente ensinadas, reconhecemos que a carga da exigência e a aquisição de novas habilidades podem ser intimidantes para pessoas que são novas neste mundo.
+O *The Turing Way* é um manual para apoiar os alunos, seus supervisores, financiadores e editores de periódicos a garantir que a ciência de dados reproduzível seja "fácil demais para não ser feita", mesmo para pessoas que nunca trabalharam dessa forma antes.
+Ele inclui o material de treinamento sobre controle de versão, teste de análise e comunicação aberta e transparente com usuários futuros, além de estudos de caso e workshops do Instituto Turing(*Turing Institute*).  
+Este projeto é desenvolvido abertamente e todas e quaisquer perguntas, comentários e recomendações são bem-vindas em nosso repositório do GitHub: <https://github.com/alan-turing-institute/the-turing-way>.
+
+### O time
+
+Esta é uma parte do trabalho de planejamento da equipe do projeto no Turing Institute.  
+Para mais informações sobre como nos contactar, consulte o [documento com as formas de trabalho](ways_of_working.md).
+
+![Foto do time](book/website/figures/TuringWayTeam.jpg)
+
+### Como contribuir
+
+:construction: Este repositório é sempre um trabalho em andamento e **todos** são incentivados a nos ajudar a construir algo que seja útil para muitos. :construction:
+
+Todos são convidados a seguir nosso [código de conduta](CODE_OF_CONDUCT.md) e a checar nossas [diretrizes de contribuição](CONTRIBUTING.md) para obter mais informações sobre como começar.
+
+Se você não está familiarizado ou confiante em contribuir com o GitHub, também pode contribuir com um estudo de caso e com suas dicas e truques por meio de nosso [formulário Google](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+
+### Citando o _The Turing Way_
+
+Você pode fazer referência ao _The Turing Way_ através do arquivo Zenodo do projeto usando DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
```

Comment:
Você pode referenciar o _The Turing Way_ através do arquivo Zenodo do projeto usando DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).

Reply:
They're synonyms. I don't see any difference in the meaning or in how it sounds. They're pretty the same.